client/src/main/java/io/pravega/client/stream/StreamCut.java

```diff
@@ -54,6 +64,13 @@ private Object readResolve() {
      * Serializes the cut to a compact byte array.
      */
     ByteBuffer toBytes();
+
+    /**
+     * Serializes the cut to a compact base64 string representation.
+     * @return Base64 representation of the StreamCut.
+     * @throws SerializationException Exception during serialization.
+     */
+    String asText() throws SerializationException;
```

Comment:
the implementation of `asText` is left to implementer of this interface while `of` is implemented here. 
Both logic should coexist.

Reply:
done.